test/mobile/test_upgrader_bytecode_table_example.h

```diff
@@ -0,0 +1,329 @@
+#pragma once
+/**
+ * Generated by gen_mobile_upgraders.py
+ */
+
+#include <ATen/core/ivalue_inl.h>
+#include <torch/csrc/jit/mobile/code.h>
+#include <torch/csrc/jit/mobile/function.h>
+#include <torch/csrc/jit/serialization/import_export_functions.h>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace c10 {
+TypePtr parseType(const std::string& pythonStr);
+} // namespace c10
+
+namespace torch {
+namespace jit {
+
+struct Instruction;
+struct Upgrader {
+  int min_version;
+  int max_version;
+  std::string upgrader_name;
+  int index;
+};
+
+// From operator_versions.yaml
+
+const std::unordered_map<std::string, std::vector<Upgrader>> kOperatorVersionMap(
```

Comment:
Can this be a different name if we are using kOperatorVersionMap for the actual version_map.h

Reply:
Yeah I will rename it